### PR TITLE
chore: switch default branch to main

### PR DESCRIPTION
**Pull request by jakemhiller**
Switching all master references before switching the branch over

### CODE REVIEW

File: packages/gamut-labs/CHANGELOG.md [hunk 1]
```diff
@@ -1,116 +1,100 @@
 ### [1.8.3](https://github.com/Codecademy/client-modules/compare/@codecademy/gamut-labs@1.8.0...@codecademy/gamut-labs@1.8.3) (2020-07-29)
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/CHANGELOG.md:1]: Revert changes in this file?

Comment (jakemhiller) [reply packages/gamut-labs/CHANGELOG.md:1]: weird, you mean the formatting changes? honestly i don't know if these changes will really matter, they're going to be overwritten by the changelog task and probably still contain the old default branch at least in terms of the commit messages

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/CHANGELOG.md:1]: Err, this kind of file? (read: changelogs)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f21c64b9bc04de8713946ab--gamut-preview.netlify.app>https://5f21c64b9bc04de8713946ab--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f21c64b9bc04de8713946ab>Deploy Logs</a></p>

**Pull request merged**